solr-ingest/src/main/java/edu/unc/lib/dl/data/ingest/solr/indexing/DocumentIndexingPackage.java

```diff
@@ -205,25 +207,27 @@ public void setChildren(List<PID> children) {
 		this.children = children;
 	}
 
-	public Map<String, List<String>> getTriples() throws IndexingException {
+	public Model getTriples(String path) throws IndexingException {
 		if (triples == null) {
-			triples = loader.loadTriples(this);
+			triples = loader.getRepository().getContentObject(path).getModel();
```

Comment:
It looks like this was just part of a leftover early attempt at implementing the setdatesfilter stuff, before we knew we wouldn't need the getTriples() anymore.